app/models/daily_report.rb

```diff
@@ -17,15 +17,10 @@ def posts(sort='', page=1, per_page=25)
           then posts.created_at
           else coalesce(min(replies_today.created_at), posts.created_at)
           end as first_updated_at")
-      .joins(:board)
```

Comment:
You're getting errors because you're including this join in the posts_from_relation call in the ApplicationController now, but the rspec test is just testing DailyReport.posts so it looks for this, fails to find it because we haven't called posts_from_relation, and crashes.